tests/spec/core/validators-spec.js

```diff
@@ -30,6 +31,13 @@ describe("Core - Validators", () => {
     it("generates an error if the element name is not valid", () => {
       const elements = ["my element", "crypto$", "🪳", "-something", ""];
       for (const element of elements) {
+        sub(
```

Comment:
We should probably use a spy here, and/or use jasmine's `toHaveBeenCalledWith`

Reply:
Nearly went there, but this seemed cleaner to me... the spy adds a lot of indirection.

Reply:
Got rid of the actual test part, as it's not important for these.

Reply:
Plus a lot of these tests still check "respec-offending-element" :( will need to rewrite them later.